### DISCUSSION TITLE
<title>Not showing commits'

### DISCUSSION THREAD

**Discussion by vahideabdi** [community/community#73608] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Oftentimes, when I pushed my contribution to the git, it will appear on the day before the exact day I pushed it on my git graph.

I mean, I push my commits today but they appear on my GitHub as I push the commits yesterday. Please how can I troubleshoot this issue?

**Comment by mehulrajdev** [comment, marked as answer]
Check if your computer time zone is properly set, that might cause this issue. Every commit has its own timestamp and it includes time zone so if you are using a service to commit for your, the commits might appear on the wrong day.
